mappings/net/minecraft/entity/ai/goal/UniversalAngerGoal.mapping

```diff
@@ -0,0 +1,11 @@
+CLASS net/minecraft/class_5398 net/minecraft/entity/ai/goal/UniversalAngerGoal
```

Comment:
Also there is another non-final instance int field which is the last recorded "lastAttackedTime" for the owner entity